pycbc/waveform/compress.py

```diff
@@ -413,6 +419,9 @@ def compress_waveform(htilde, sample_points, tolerance, interpolation,
                 outptr += 2;
                 findex++;
             }
+            if (next_sfindex == hlen){
+            break;
+            }
         }
```

Comment:
This `if (...) break` should go outside of the while loop (inside the for loop); i.e., it should happen after the `}` at line 425, not before. I addressed this in PR #1491, but I think it's best to fix it here. Please also make the indentation of the if be the same as the block inside the for loop, to make it clear which loop the break is in.